.ci/Jenkinsfile

```diff
@@ -317,12 +317,17 @@ def generateFunctionalTestStep(Map args = [:]){
   }
   tags += excludeNightlyTag
 
+  if (isPR() || isUpstreamTrigger(filter: 'PR-')) {
```

Comment:
https://github.com/elastic/e2e-testing/pull/883 has not been backported, so, do we need it?

Reply:
No, let's discard 6.8.x branch, as per #879